@dotnet/dnceng - I am getting package failures even on retry:

> root/runtime/src/installer/pkg/projects/netcoreapp/sfx/Microsoft.NETCore.App.SharedFx.sfxproj(0,0): error NU1102: Unable to find package Microsoft.NETCore.Platforms with version (>= 5.0.0-ci.19620.1)
  - Found 1 version(s) in /root/runtime/artifacts/packages/Release/Shipping/ [ Nearest version: 5.0.0-ci.19619.1 ]
  - Found 0 version(s) in /root/runtime/artifacts/packages/Release/NonShipping/

--- a/src/coreclr/tests/src/JIT/Regression/JitBlue/GitHub_27924/GitHub_27924.cs
+++ b/src/coreclr/tests/src/JIT/Regression/JitBlue/GitHub_27924/GitHub_27924.cs
@@ -15,7 +15,8 @@ class Program
 
     static void Work()
     {
-        for (uint i = 0; i < 1000000; i++) {
+        for (uint i = 0; i < 1000000; i++)
+        {
             var a = s[i++ % s.Length];
 
             ref byte p = ref a[0];
@@ -37,12 +38,12 @@ class Program
         List<Task> tasks = new List<Task>();
         for(int i = 0; i < 5; i++)
         {
-            // new Thread(Work).Start();
             tasks.Add(Task.Run(Work));
         }
 
         Random r = new Random();
-        for (uint i = 0; i < 10000; i++) {
+        for (uint i = 0; i < 10000; i++)
+        {
             s[r.Next(s.Length)] = new byte[3 + r.Next(100)];
         }
         Task t = Task.WhenAll(tasks);

